common/appointment/update.ts

```diff
@@ -115,7 +115,7 @@ export async function updateAppointment(
         }
     }
     const zoomUpdate = {
-        start: newStart,
+        startTime: newStart,
```

Comment:
Huh, why didn't this even compile?

Reply:
Aah cause the type is not specified and startTime is optional and objects can have additional props :/